src/packages.builds

```diff
@@ -32,5 +32,6 @@
     <GenerateNetStandardSupportTable Reports="@(Reports)" DocFilePath="$(ProjectDir)Documentation\architecture\net-platform-standard.md" InsertIntoFile="True" />
   </Target>
 
+  <Import Project="$(ToolsDir)versioning.targets" Condition="Exists('$(ToolsDir)versioning.targets')" />
```

Comment:
This would get included if you included dir.targets which imports dir.traversal.targets and build.common.targets.  These "src/*.builds" files don't include them.  We made a similar change for "src/tests.builds"